scripts/sync.py

```diff
@@ -147,7 +154,12 @@ def do(args):
 
     # transfer new files after all sync attempts are done
     if args.lochness_sync_send:
-        lochness_to_lochness_transfer(Lochness)
+        if args.s3:
+            lochness_to_lochness_transfer_s3(Lochness)
+        elif args.rsync:
+            lochness_to_lochness_transfer_rsync(Lochness)
+        else:
```

Comment:
What argument is triggering the `else` block?

Reply:
`sftp` transfer is triggered in the `else` block